### PR TITLE
[3.9] Adjustments for upcoming Install from Web plugin update

### PR DESCRIPTION
**Pull request by mbabker**
### Summary of Changes

In preparation for the [upcoming Install from Web update](https://github.com/joomla-extensions/install-from-web-client/releases/tag/2.0.0-alpha) there are language string changes needed, and since the language file is in this repo and not distributed as part of the plugin, the updates must be made here.

In short, this:

- Marks strings being removed deprecated, they have to stay here since the language file will be used for any IFW plugin release
- Adds new strings replacing hardcoded English, similar to the changes in 4.0
- Adds the media path to `.gitignore`

### Testing Instructions

Install IFW plugin update, apply this patch.  Without it you're going to have untranslated strings in a couple JavaScript actions.

### CODE REVIEW

File: administrator/language/en-GB/en-GB.plg_installer_webinstaller.ini [hunk 1]
```diff
@@ -4,9 +4,18 @@
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_INSTALLER_WEBINSTALLER="Installer - Install from Web"
+PLG_INSTALLER_WEBINSTALLER_CANNOT_INSTALL_EXTENSION_IN_PLUGIN="This extension cannot be installed via the install from web system. Please visit the developer's website to purchase/download."
+PLG_INSTALLER_WEBINSTALLER_ERROR_PLUGIN_INCLUDED_IN_CORE="Plugin install aborted, the Install from Web plugin is included in core as of Joomla! 4.0."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.plg_installer_webinstaller.ini]: Plugin installation has been aborted. The ..

File: administrator/language/en-GB/en-GB.plg_installer_webinstaller.ini [hunk 2]
```diff
@@ -4,9 +4,18 @@
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_INSTALLER_WEBINSTALLER="Installer - Install from Web"
+PLG_INSTALLER_WEBINSTALLER_CANNOT_INSTALL_EXTENSION_IN_PLUGIN="This extension cannot be installed via the install from web system. Please visit the developer's website to purchase/download."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.plg_installer_webinstaller.ini]: Using not via

File: administrator/language/en-GB/en-GB.plg_installer_webinstaller.ini [hunk 3]
```diff
@@ -4,9 +4,18 @@
...
+PLG_INSTALLER_WEBINSTALLER_ERROR_PLUGIN_INCLUDED_IN_CORE="Plugin install aborted, the Install from Web plugin is included in core as of Joomla! 4.0."
+; This string is not used.
 PLG_INSTALLER_WEBINSTALLER_LOAD_APPS="Display the extensions"
```
Comment (zero-24) [inline administrator/language/en-GB/en-GB.plg_installer_webinstaller.ini]: can't we drop that string than when it is not used in the 1.x version too?

Comment (mbabker) [reply administrator/language/en-GB/en-GB.plg_installer_webinstaller.ini]: Strings in this repo can only be deprecated to be removed at the next major version.

Comment (zero-24) [reply administrator/language/en-GB/en-GB.plg_installer_webinstaller.ini]: Fine for me.

**Review by brianteeman — CHANGES_REQUESTED** [review]

File: administrator/language/en-GB/en-GB.plg_installer_webinstaller.ini (hunk 1 above)
Comment (mbabker) [reply administrator/language/en-GB/en-GB.plg_installer_webinstaller.ini]: Changed

File: administrator/language/en-GB/en-GB.plg_installer_webinstaller.ini (hunk 2 above)
Comment (mbabker) [reply administrator/language/en-GB/en-GB.plg_installer_webinstaller.ini]: Changed

**Review by brianteeman — APPROVED** [review]

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 073db4badaea4c762f5d0b45071016501bfb42fc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21950">issues.joomla.org/tracker/joomla-cms/21950</a>.</sub>

**Comment by zero-24** [comment]
Merging. Thanks!

**Pull request merged**